modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_mini/transition.js

```diff
@@ -41,12 +39,13 @@ export default function ({namespace}) {
 			`${namespace}transitionComments`
 		);
 
+		transitionComments.classList.remove('hide');
+
 		transitionComments.style.display = 'block';
 
 		const dialog = Liferay.Util.Window.getWindow({
```

Comment:
@kresimir-coko , it looks like I am no longer able to use Liferay.Util.Window.getWindow as a function (no longer exists?). I get following error:`Uncaught TypeError: Liferay.Util.Window.getWindow is not a function`. I am requesting your assistance in refactoring the dialog here as front-end is not my strongest point. Could you let me know best way to proceed?
`

Reply:
Hello @kresimir-coko , I believe I did most of the conversion correctly. Please correct me if the following needs improvement. I have one question for how to replace `bodyContent: form` correctly?
```
		openModal
			({
				buttons: [
				{
					displayType: 'primary',
					label: Liferay.Language.get('done'),
					onClick() {
						{
							submitForm(form);
						}
					},
				},
				{
					autoFocus: true,
					displayType: 'secondary',
					label: Liferay.Language.get('cancel'),
					type: 'cancel',
				},
			],
			height: '40vh',
			id: `${namespace}transitionFm`,
			size: 'lg',
			title: link.innerText
		});
	});

```

Reply:
> it looks like I can no longer use `Liferay.Util.Window.GetWindow` 

There’s 25 usages of this in the codebase, it wasn’t removed, so it should work like it did before.

[This test demonstrates](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-web/test/liferay/modal/Modal.js#L106-L110) that if you are passing it a `bodyComponent` prop, it needs to be a React component. 

Other option is to use the `bodyHTML` prop and pass HTML to it, maybe you could try using that instead of `bodyContent: form` and replace it with `bodyHTML: form`.

I am not an expert here, and I haven't migrated that many of these to know how it all works without testing it in the code myself. But, it still looks to me that you shouldn't be getting the `Uncaught TypeError: Liferay.Util.Window.getWindow is not a function` error and you might have a different problem with your code.